files/docker/node/dockerfile_stage3alpha

```diff
@@ -0,0 +1,110 @@
+    && locale-gen \
+    && echo "export LC_ALL=en_US.UTF-8" >> ~/.bashrc \
+    && echo "export LANG=en_US.UTF-8" >> ~/.bashrc \
+    && echo "export LANGUAGE=en_US.UTF-8" >> ~/.bashrc
+
+ENV \
+    ENV=/etc/profile \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US.UTF-8 \
+    USER=guild \
+    CNODE_HOME=/opt/cardano/cnode \
+    PATH=/home/guild/.local/bin:/nix/var/nix/profiles/per-user/guild/profile/bin:/nix/var/nix/profiles/per-user/guild/profile/sbin:/opt/cardano/cnode/scripts:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/home/guild/.cabal/bin \
+    GIT_SSL_CAINFO=/etc/ssl/certs/ca-certificates.crt \
+    NIX_SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt \
+    NIX_PATH=/nix/var/nix/profiles/per-user/guild/channels
+
+# PREREQ --no-install-recommends
+RUN apt-get update && apt-get install -y libcap2-bin ncurses-bin iproute2 curl wget apt-utils xz-utils netbase sudo coreutils dnsutils net-tools procps cron tcptraceroute bc 
+    
+ADD https://raw.githubusercontent.com/stakelovelace/cardano-node/master/crontab /etc/cron.d/crontab
+RUN chmod 0644 /etc/cron.d/crontab && touch /var/log/cron.log 
+
+RUN cd /usr/bin \
+&& sudo wget http://www.vdberg.org/~richard/tcpping \
+&& sudo chmod 755 tcpping 
+
+# SETUP Guild USER 
+RUN adduser --disabled-password --gecos '' guild \
+&& echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers \
+&& adduser guild sudo \ 
+&& chown -R guild:guild /home/guild/.* 
+
+USER guild
+WORKDIR /home/guild
+
+# INSTALL NIX
+RUN sudo curl -sL https://nixos.org/nix/install | sh \
+    && sudo ln -s /nix/var/nix/profiles/per-user/etc/profile.d/nix.sh /etc/profile.d/ \
+    && . /home/guild/.nix-profile/etc/profile.d/nix.sh \
+    && sudo crontab -u guild /etc/cron.d/crontab
+
+# INSTALL DEPS  
+RUN /nix/var/nix/profiles/per-user/guild/profile/bin/nix-channel --update \
+    && /nix/var/nix/profiles/per-user/guild/profile/bin/nix-env -i python3 systemd libsodium tmux jq ncurses libtool autoconf git wget gnupg column less openssl vim \
+    && /nix/var/nix/profiles/per-user/guild/profile/bin/nix-env -u --always \
+    && /nix/var/nix/profiles/per-user/guild/profile/bin/nix-collect-garbage -d 
+
+# GUILD SKAffold
+RUN sudo mkdir -p $CNODE_HOME/files $CNODE_HOME/db $CNODE_HOME/db/blocks $CNODE_HOME/logs $CNODE_HOME/scripts $CNODE_HOME/sockets $CNODE_HOME/priv/files \
+    && echo "export PATH=/nix/var/nix/profiles/per-user/guild/profile/bin:/nix/var/nix/profiles/per-user/guild/profile/sbin:/opt/cardano/cnode/scripts:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/home/guild/.cabal/bin" >> ~/.bashrc \
+    && echo "alias cntools=/opt/cardano/cnode/scripts/cntools.sh" >> ~/.bashrc \
+    && echo "alias gLiveView=/opt/cardano/cnode/scripts/gLiveView.sh" >> ~/.bashrc
+
+# HYDRA CFG
+ADD https://hydra.iohk.io/build/3670619/download/1/mainnet-shelley-genesis.json $CNODE_HOME/priv/files/
+ADD https://hydra.iohk.io/build/3670619/download/1/mainnet-byron-genesis.json $CNODE_HOME/priv/files/
```

Comment:
Do we need hydra config? There are a lot of differences in the two configs

Reply:
To be honest I used those because the official one... but I can use the guild one for mainnet

Reply:
you commented the genesys lines and I got confused :)

Reply:
Oops my bad yeah it was for next lines, it was for config instead of genesis indeed 🙁